regexing/src/main/java/Validation.java

```diff
@@ -68,7 +83,9 @@ public boolean zipCode(String input){
      * @return
      */
     public boolean email(String input){
-        return true;
+        Pattern p = Pattern.compile("\\w+\\.?\\w+@\\w+\\.\\w+");
```

Comment:
Another thing to think about here is that you might have multiple . in the front part so might be a bit more flexible to have [\w\.]+ or something similar though this works fine.